projects/memapCore/src/main/java/memap/messages/planning/StorageMessage.java

```diff
@@ -24,6 +24,7 @@
 	public double efficiencyDischarge;
```

Comment:
I extended the storage message to contain an option to clarify the losses dimension. We can use that later on to calculate the hourly losses precisely.